build.gradle

```diff
@@ -276,10 +293,6 @@ processResources {
     }
 }
 
-loom {
-    splitEnvironmentSourceSets()
```

Comment:
This was before `sourceSets` for a reason, this probably breaks stuff

Reply:
works fine in my IDE. ``net.minecraft.client`` isn't available from main sourceSet, but it is available in ``client`` sourceset.